pallets/parachain-staking/src/lib.rs

```diff
@@ -484,6 +490,17 @@ pub mod pallet {
 	pub(crate) type CandidateInfo<T: Config> =
 		StorageMap<_, Twox64Concat, T::AccountId, CandidateMetadata<BalanceOf<T>>, OptionQuery>;
 
+	/// Stores outstanding delegation requests per collator.
+	#[pallet::storage]
+	#[pallet::getter(fn delegation_scheduled_requests)]
+	pub(crate) type DelegationScheduledRequests<T: Config> = StorageMap<
+		_,
+		Blake2_128Concat,
```

Comment:
This is probably fine as a cheaper hash given that (1) the user has put up some minimum amount of funds, (2) they aren't receiving rewards, and (3) someone could revoke this for them at any point in time.

But I'm also fine with delegation reducing being more expensive than usual...

Reply:
But given that the key is controlled by the user, the docs recommend using a cryptographic secure hash.
Isn't that the case here?

Reply:
Yes, but there is an economic barrier to them abusing it. We can revisit it later, although it would require a migration at that point. It's not a huge deal, but it does add cost anything related to scheduled requests.